samples/powerpoint/text/get-set-textrange.yaml

```
@@ -0,0 +1,180 @@
+order: 1
+id: powerpoint-get-set-textrange
```

Consider removing id then rerunning `yarn start`
```suggestion
id: powerpoint-text-get-set-textrange
```